.circleci/generate.sh

```diff
@@ -18,7 +18,11 @@
 #
 
 BASEDIR=`dirname $0`
-BASE_BRANCH=cassandra-3.11
+
+# Allow the user to set an env variable to control this, but default to our version branch
+[ -n "${BASE_BRANCH}" ] || {
```

Comment:
Do we have a use case for setting the branch trough a previously set env var? I think no other options work that way. If we don't have a use case we can save us the additional complexity of having two ways of setting the branch and simplify both the help and error messages.

Reply:
I can remove it. My setup auto-sets env vars for a given work dir, so it's one less flag to have to remember to place